packages/components/src/color-edit/index.js

```diff
@@ -68,22 +70,22 @@ function ColorOption( {
 					<ColorIndicatorStyled colorValue={ color.color } />
 				</FlexItem>
 				<FlexItem>
-					{ isEditing ? (
+					{ isEditing && ! canOnlyChangeValues ? (
 						<ColorNameInput
 							value={ color.name }
 							onChange={ ( nextName ) =>
 								onChange( {
 									...color,
 									name: nextName,
-									slug: kebabCase( nextName ),
+									slug: slugPrefix + kebabCase( nextName ),
```

Comment:
Ah, I see. We need to fix the kebabCase import to `import { kebabCase } from 'lodash';`.